index.html

```diff
@@ -26,7 +27,7 @@
   <section class="guessStatement">
     <p class="recall">Your last guess was:</p>
     <div class="populate"></div>
-    <span class="message"></span>
+    <div class="message"></div>
```

Comment:
`div`, or some other block element, is a better choice than an inline `span` element.